etc/ProdOfflineConfiguration.py

```diff
@@ -639,7 +645,21 @@
                dqm_sequences=["@common"],
                scenario=ppScenario)
 
-# Will be "JetMET"  after  HLT V1.3 menu
+DATASET ["JetMET"]
+
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=True,
+               write_dqm=True,
+               tape_node="T1_UK_RAL_MSS",  # JetHT was in "T1_UK_RAL_MSS" , MET was in "T1_DE_KIT_MSS"
+               disk_node="T1_UK_RAL_Disk", # JetHT was in "T1_UK_RAL_Disk", MET was in "T1_DE_KIT_Disk"
+               alca_producers=["HcalCalIsoTrkProducerFilter", "TkAlJetHT", "HcalCalNoise"],
+               dqm_sequences=["@common", "@jetmet", "@L1TMon", "@hcal"],
+               physics_skims=["EXODelayedJetMET", "JetHTJetPlusHOFilter", "LogError", "LogErrorMonitor"],
+               timePerEvent=5.7,  # copied from JetHT - should be checked
+               sizePerEvent=2250, # copied from JetHT - should be checked
```

Comment:
Again these settings come from JetHT PD, while MET PD was using the default values...how do we set them@germanfgv @jhonatanamado @jeyserma @drkovalskyi ?

Reply:
For this I don't really know. Do we have any estimates on how big the events will be?. We will have to tweak them after we process the runs with them.